spec/classes/server/conf_spec.rb

```diff
@@ -9,10 +9,11 @@
       end
 
       context "on os #{os}" do
+        # This is a common dependency that is notified
+        let(:pre_condition){ 'service { "sshd": }' }
 
         context 'with default parameters, openssh_version=5.3, both simp_options::fips and fips_enabled false' do
           let(:facts) { os_facts.merge( { :openssh_version => '5.3', :fips_enabled => false } ) }
-          let(:pre_condition){ 'include "::ssh"' }
```

Comment:
How can this test of class with assert_private() work without  the pre_condition?

Reply:
No idea, but it certainly does :man_shrugging: